azkaban-common/src/main/java/azkaban/trigger/TriggerManager.java

```diff
@@ -61,10 +63,13 @@
 
   private String scannerStage = "";
 
+  // TODO kunkun-tang: Before apply guice to this class, we should make
+  // ExecutorManager guiceable.
   public TriggerManager(Props props, TriggerLoader triggerLoader,
       ExecutorManager executorManager) throws TriggerManagerException {
 
-    this.triggerLoader = triggerLoader;
+    // TODO kunkun-tang: Doing hack here to allow calling new azkaban-db code. Should fix in future.
+    this.triggerLoader = ServiceProvider.SERVICE_PROVIDER.getInstance(TriggerLoader.class);
```

Comment:
I tried Inject this constructor, but didn't work. I believe Executor Manager still can not be imported by guice now. @suvodeep-pyne